### PR TITLE
Lint and drop missing autocomplete call.

### PR DESCRIPTION
**Pull request by gburtini**
https://github.com/replit/repl-it-web/pull/28132

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,74 +1,63 @@
...
-	return new LanguageSupport(
-		csharpLanguage,
-		csharpLanguage.data.of({
-			autocomplete: ifNotIn(
-				["LineComment", "BlockComment"]
-			),
-		})
-	);
+  return new LanguageSupport(csharpLanguage);
 }
```
Comment (gburtini) [inline src/index.ts:63]: Material change is here. [ifNotIn](https://github.com/codemirror/autocomplete/blob/main/src/completion.ts#L139) requires an argument for an autocompletion source, which we don't have here. 

The rest is linter and unused references.

Comment (sergeichestakov) [reply src/index.ts:63]: weird how did we not catch this before? also how does this pass TS?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@replit/codemirror-lang-csharp",
   "description": "A C# mode for CodeMirror 6",
-  "version": "6.0.0",
+  "version": "6.1.0",
   "author": {
```
Comment (sergeichestakov) [inline package.json]: let's bump in separate commit?

**Review by sergeichestakov — APPROVED** [review]
thanks!

**Comment by gburtini** [comment]
It does not pass TS.

File: src/index.ts (hunk 1 above)
Comment (gburtini) [reply src/index.ts:63]: It does not pass TS.

File: package.json (hunk 2 above)
Comment (gburtini) [reply package.json]: OK.

**Pull request merged**